porfile.html

```diff
@@ -56,6 +56,196 @@ <h1 class="title">
         </div>
 
     </section>
+    <section id="projects">
```

Comment:
Great work @ogoigbe12  you implemented almost all this project requirements perfectly 💪🎉, Only one small thing to mention before approving your work , you need to use grid layout to align you project section container as requested in the requirements :
"In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card."